ui/src/translations/en.json

```diff
@@ -135,5 +136,10 @@
   "please_refer_to": "Please refer to",
   "import_solution_hint": "You should import a solution to prepare your environment",
   "redirecting": "Redirecting…",
-  "documentation": "Documentation"
+  "documentation": "Documentation",
+  "env_preparation": "Environment preparation",
+  "env_preparation_success": "Environment {envName} is being prepared.",
+  "env_preparation_failed": "Environemnt {envName} preparation failed.",
+  "preparing_environemnt": "preparing the environment",
```

Comment:
ok. At the beginning, I would like to have this prop used for the text of the loader.
But yeah, it makes sense to reuse`env_preparation_success`

Reply:
Oh OK I see! So no, let's keep a loading message as this one.

However, `env_preparation_success` should show a "successful" status, no? If yes, maybe reword to show:
- in English: `Environment {envName} preparation succeeded.`
- in French: `La préparation de l'environnement {envName} a réussi.`

The current message for `_success` is actually meaning something is `_in_progress` ;)

Reply:
Ok. For the loading message, I will keep the previous one.

However, I don't think it's a good idea to change the text of `env_preparation_success`. Probably we should change the key to `env_preparation_in_progress` though.

I am going to give the context. The text will be displayed as the notification after the Salt job is done. So in theory, we couldn't say it's already succeeded because we still need to check the deployment of the solution.

So IMO, `Environment {envName} is being prepared.` is better. What do you think?

Reply:
> The text will be displayed as the notification after the Salt job is done. So in theory, we couldn't say it's already succeeded because we still need to check the deployment of the solution.

That's where I disagree ;) When Salt returns after the `prepare-environment`, it means (at least, from Salt PoV) that the env has been prepared.
It may not mean that all resources (UI/Operator) are available in the current implem, but I'd rather add this check on Salt side than needing an extra step in UI or CLI.